docs/docs/guides/migrate/introduction.md

```diff
@@ -76,7 +75,7 @@ In case all your applications depend on ZITADEL after the migration date, and ZI
 For all other cases, we recommend that the **legacy system orchestrates the migration** of users to ZITADEL for more flexibility:
 
 - Update your legacy system to create a user in ZITADEL on their next login, if not already flagged as migrated, by using our APIs (you can set the password and a verified email)
-- Redirect migrated users with a login hint in the [auth request](/docs/apis/openidoauth/authrequest.mdx) to ZITADEL to pre-select the user
+- Redirect migrated users with a login hint in the [auth request](/docs/oidc-playground) to ZITADEL to pre-select the user
```

Comment:
Are you sure you want to change that one?

Reply:
this should be fine. it is /playgrounds/oidc now